components/static.html

```diff
@@ -80,20 +180,8 @@ <h2 class="md-title">MyUW works best in Firefox and Chrome</h2>
 </script>
 
 <!-- Loading state -->
-<div ng-hide="portal.loading.hidden" ng-class="{slowfade: portal.loading.startFade}" style="
-        position : fixed;
-        top: 0;
-        z-index: 2000;
-        color: darkgrey;
-        background: #EAE8DF;
-        height: 100vh;
-        width: 100vw;
-        text-align: center;
-        padding-top: 45vh;
-        font-size: xx-large;
-        margin: -8px;
-        font-family: Roboto, Helvetica Neue, sans-serif;"
-      id="loading-splash">
+<h1 class="sr-only">The application is getting ready...</h1>
```

Comment:
Looks like, for screen readers, this `<h1` will remain as the apparent top heading on the page even after the application has gotten as ready as it's going to get. Should this h1 have an `ng-if` or `ng-hide` or so so that it will be removed from the page when `portal.loading.hidden`?

<img width="1068" alt="Screen Shot 2019-04-29 at 1 26 44 PM" src="https://user-images.githubusercontent.com/952283/56918107-c7175a80-6a82-11e9-9fe3-3db80de5a133.png">

Reply:
Fixed in https://github.com/uPortal-Project/uportal-app-framework/pull/907